taxi/views.py

```diff
@@ -87,3 +88,26 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
```

Comment:
The task requires using `UserCreationForm` for creating users, but the current implementation uses `DriverForm`. Please update this to use `UserCreationForm` to meet the task requirements.